### PR TITLE
Revert "Revert "[TEST] test XLA CI w/ SymInts""

### PR DESCRIPTION
**Pull request by miladm**
Reverts pytorch/xla#3651

reverting the reverted changes for symint

### CODE REVIEW

**Review by Krovatkin — APPROVED** [review]

**Comment by miladm** [comment]
Looks like this PR is blocked on a reverted PR in PyTorch. Waiting on @Krovatkin to unblock before we merge.

File: torch_xla/csrc/tensor_impl.h
Comment (miladm) [inline torch_xla/csrc/tensor_impl.h:34]: @Krovatkin looks I no longer see the `virtual c10::SymIntArrayRef sym_sizes()` [here](https://github.com/pytorch/pytorch/blob/master/torch/csrc/lazy/core/tensor_impl.h#L46). Is that a final call or do you plan to add it back?

**Comment by Krovatkin** [comment]
@miladm this should be unblocked now!

**Review by JackCaoG — APPROVED** [review]

**Pull request merged**
